packages/teams-js/src/public/teamsAPIs.ts

```diff
@@ -36,7 +36,7 @@ export namespace teamsCore {
    * default print handler
    */
   export function print(): void {
-    window.print();
+    typeof window !== undefined ? window.print() : new Error('window does not exist');
```

Comment:
It looks like if window is not defined you are just creating an error and doing nothing with it. You probably want to throw the error?

Reply:
Yeah, I'm dumb. Should be fixed!

Reply:
Should this code also never run because it only exists for the SSR case? If so maybe the message of the `Error` should be more explicit. E.g., "This codepath only exists to enable compilation in an SSR environment. If this error has actually been thrown something very wrong has happened and it is a bug."